static/app/utils/queryClient.tsx

```diff
@@ -49,6 +49,17 @@ export type ApiQueryKey =
         Record<string, any>
       >,
     ];
+export type InfiniteApiQueryKey =
+  | readonly ['infinite', url: string]
```

Comment:
added this const as a prefix because: most generic to most specific is the best order.